hosting/src/firebase.ts

```diff
@@ -15,12 +15,12 @@ export const app = initializeApp({
 });
 
 const emulatorDetails = {
-	authURL: process.env.REACT_APP_FIREBASE_AUTH_EMULATOR_URL,
-	firestoreHost: process.env.REACT_APP_FIREBASE_FIRESTORE_EMULATOR_HOST,
-	firestorePort: process.env.REACT_APP_FIREBASE_FIRESTORE_EMULATOR_PORT
+	authURL: import.meta.env.VITE_FIREBASE_AUTH_EMULATOR_URL,
+	firestoreHost: import.meta.env.VITE_FIREBASE_FIRESTORE_EMULATOR_HOST,
+	firestorePort: import.meta.env.VITE_FIREBASE_FIRESTORE_EMULATOR_PORT
 };
 
-if (process.env.REACT_APP_ENV !== "production") {
+if (import.meta.env.DEV) {
```

Comment:
How does this work? Previously we were setting the `REACT_APP_ENV` to `production` as part of the CI flow:
https://github.com/codesupport/code-review-tool/blob/dd3ffa174e312737b7936ce4fbdf544898ae482f/.github/workflows/main.yml#L12

Do we need to do something similar here? Does any of the CI flow need changing?

Reply:
It does need changing! I removed that line there, since it's not needed anymore :)

During build, vite will automatically detect this and strip out the code. Here is a match from the bundle created using `npm run build` on the left and the original code on the right:

![grafik](https://github.com/codesupport/code-review-tool/assets/26303198/a248d5a2-aed0-4915-80b1-48cf48778183)

During build (implicitly for production) `import.meta.env.DEV` is false, so Vite trims out the whole `if` statement and even drops `emulatorDetails` since it's not referenced anymore after that.

Reply:
Oh that's excellent!